Gosu/Window.h

```diff
@@ -31,11 +47,11 @@ Gosu_Window* Gosu_Window_create(int width, int height, int fullscreen, double up
 // callbacks
 void Gosu_Window_update(Gosu_Window* window, void function());
 void Gosu_Window_draw(Gosu_Window* window, void function());
-void Gosu_Window_button_down(Gosu_Window* window, void function());
-void Gosu_Window_button_up(Gosu_Window* window, void function());
-void Gosu_Window_drop(Gosu_Window* window, void function());
-void Gosu_Window_needs_redraw(Gosu_Window* window, void function());
-void Gosu_Window_needs_cursor(Gosu_Window* window, void function());
+void Gosu_Window_button_down(Gosu_Window* window, void function(unsigned btn));
+void Gosu_Window_button_up(Gosu_Window* window, void function(unsigned btn));
+void Gosu_Window_drop(Gosu_Window* window, void function(const char* filename));
+void Gosu_Window_needs_redraw(Gosu_Window* window, int function());
```

Comment:
When compiled as C, `int function()` is a variadic function, equivalent to `int function(...)` in C++. `int function(void)` would be more typesafe.